config/routes.rb

```diff
@@ -3,8 +3,23 @@
   
   # allow users to only sign in and sign out
   devise_for :admins, only: :sessions
+
+  # change url for sign in 
+  devise_scope :admin do 
+    get '/sign-in', to: 'devise/sessions#new'
```

Comment:
your tabbing is not consistent across the app. here it is one tab, but in others it is two tabs. should be an easy fix.